plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/checkpoint/CheckpointSchemaManager.java

```diff
@@ -109,6 +110,13 @@ public CheckpointSchemaManager(TypeManager typeManager)
                 RowType.field("readVersion", BIGINT),
                 RowType.field("isolationLevel", VARCHAR),
                 RowType.field("isBlindAppend", BOOLEAN)));
+
+        sidecarEntryType = RowType.from(ImmutableList.<RowType.Field>builder()
+                .add(RowType.field("path", VARCHAR))
+                .add(RowType.field("sizeInBytes", BIGINT))
+                .add(RowType.field("modificationTime", BIGINT))
+                .add(RowType.field("tags", stringMap))
+                .build());
```

Comment:
```suggestion
        sidecarEntryType = RowType.from(ImmutableList.of(
                RowType.field("path", VARCHAR),
                RowType.field("sizeInBytes", BIGINT),
                RowType.field("modificationTime", BIGINT),
                RowType.field("tags", stringMap)));

```

Reply:
in sync with the other initializations from the class.

Reply:
Using builder is better because it reduces future diff when adding a new entries to the bottom.